datasrc/network.py

```diff
@@ -403,11 +403,125 @@
 		NetStringStrict("m_Reason"),
 		NetBool("m_Force"),
 	]),
-	
+
 	### ModAPI
-	
+
 	NetMessage("ModAPI_Sv_Broadcast", [
 		NetStringStrict("m_pMessage"),
 	]),
-	
+
+
+	### NetGUI
+	## Server
+	NetMessage("ModAPI_Sv_GuiRemoveElement", [
+		NetIntAny("m_Type"),
+		NetIntAny("m_ID"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiRequestData", [
+		NetIntAny("m_ID"),
+		NetIntAny("m_Type"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiUIRect", [
+		NetIntAny("m_ID"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetArray(NetIntRange("m_Color", 0, 100), 4),
+		NetIntRange("m_Corner", 0, 255),
+		NetIntRange("m_RoundingX10", 0, 500),
```

Comment:
We can use these functions to convert floats in intergers (the TW editor use them):
	inline int f2fx(float v) { return (int)(v*(float)(1<<10)); }
	inline float fx2f(int v) { return v*(1.0f/(1<<10)); }
Then, you can remove the X10 in the name.